faq.json

```
@@ -7,8 +7,9 @@
   "messengers": "Signal is recommended for most use cases. For some people who have higher or more specific threat models, Briar, Jami, or Tinfoil Chat may be be more suitable. Note that Jami should be used with a VPN or Tor. Read https://serpentsec.1337.cx/secure-messaging-choosing-a-chat-app for more information.",
   "phones": "Linux-based smartphones provide flexibility for users of desktop Linux distributions. They may also have the least telemetry of the default operating systems on consumer phones, however, they are currently at a very experimental and unpolished stage and need support. The Pinephone is the preferred option over the Librem 5 because it is more reasonably priced and makes firmware available to the user. If security and privacy from apps is the primary focus, due to Linux's lack of sandboxing, the best options are a supported iPhone (with iCloud disabled) or a Pixel phone running GrapheneOS. Most OEM Android phones come with variations of AOSP that weaken security with substandard implementations or threaten security and privacy with unneeded bloatware, while also not providing updates anymore after a short period of time. It is preferable to aim for a manufacturer that provides an OS closer to stock, meaning minimal deviation from stock Android (AOSP).",
-  "slow": "As a tremendous amount of people use the matrix.org and privacytools.io homeservers, they are often overloaded and sometimes slow to respond. You can fix this by switching to another homeserver! Good alternatives include kde.org, nitro.chat, envs.net and fairydust.space, or even running your own! Note that at time of writing, migrating to a new server requires re-registering with another account at that server, as nomadic identities haven't been implemented yet.",
+  "privacyvsanonymity": "Privacy and anonymity are different things. Roughly put, privacy is when your identity known but what you do is unknown, so therefore no one except people whom you interact with don't know what you are doing. Anonymity is when your identity is unknown but the actions you do are known, but they can not be tied to an identity.<br>To achieve anonymity, it is recommended to make use of the Tor network with Tor browser or an OS such as Tails or Whonix.",
+  "slow": "As a tremendous amount of people use the matrix.org and privacytools.io homeservers, they are often overloaded and sometimes slow to respond. You can fix this by switching to another homeserver! Good alternatives include tchncs.de, kde.org, nitro.chat, envs.net and fairydust.space, or even running your own! Note that at time of writing, migrating to a new server requires re-registering with another account at that server, as nomadic identities haven't been implemented yet.",
```

```suggestion
  "slow": "As a tremendous amount of people use the matrix.org and privacytools.io homeservers, they are often overloaded and sometimes slow to respond. You can fix this by switching to another homeserver! Good alternatives include kde.org, nitro.chat, envs.net and fairydust.space, or even running your own! Note that at time of writing, migrating to a new server requires re-registering with another account at that server, as nomadic identities haven't been implemented yet.",
```

tchncs.de is often unreliable and in my experience the admin is unresponsive.